portal-frontend/src/app/features/edit-application/parcel-details/parcel-entry/parcel-entry.component.ts

```diff
@@ -312,6 +312,13 @@ export class ParcelEntryComponent implements OnInit {
         });
       }
 
+      const pidValue = this.pid.getRawValue();
+      if (this.isCrownLand) {
```

Comment:
do we need to reflect this on API level too?

Reply:
MR was just updated 30 seconds after you commented to include API validation and tests